Configuration/EventContent/python/AlCaRecoOutput_cff.py

```diff
@@ -38,6 +38,9 @@
 ###############################################################
 # ECAL calibration with isol. electrons
 from Calibration.EcalAlCaRecoProducers.ALCARECOEcalCalIsolElectron_Output_cff import *
+from Calibration.EcalAlCaRecoProducers.ALCARECOEcalUncalIsolElectron_Output_cff import *
+from Calibration.EcalAlCaRecoProducers.ALCARECOEcalRecalIsolElectron_Output_cff import *
+
```

Comment:
Yes, started from the last 74X. But it seems for me now that I mixed it in a hurry... I'll try to fix it!